dev/CommonStyles/APITests/CommonStylesTests.cs

```diff
@@ -92,19 +98,47 @@ public void CornerRadiusFilterConverterTest()
                 var bottomRadiusGrid = (Grid)root.FindName("BottomRadiusGrid");
                 var leftRadiusGrid = (Grid)root.FindName("LeftRadiusGrid");
 
-                VerifyCornerRadius(topRadiusGrid.CornerRadius, new[] { 6, 6, 0, 0 });
-                VerifyCornerRadius(rightRadiusGrid.CornerRadius, new[] { 0, 6, 6, 0 });
-                VerifyCornerRadius(bottomRadiusGrid.CornerRadius, new[] { 0, 0, 6, 6 });
-                VerifyCornerRadius(leftRadiusGrid.CornerRadius, new[] { 6, 0, 0, 6 });
+                Verify.AreEqual(new CornerRadius(6, 6, 0, 0), topRadiusGrid.CornerRadius);
+                Verify.AreEqual(new CornerRadius(0, 6, 6, 0), rightRadiusGrid.CornerRadius);
+                Verify.AreEqual(new CornerRadius(0, 0, 6, 6), bottomRadiusGrid.CornerRadius);
+                Verify.AreEqual(new CornerRadius(6, 0, 0, 6), leftRadiusGrid.CornerRadius);
             });
         }
 
-        private void VerifyCornerRadius(CornerRadius radius, int[] expectedValue)
+        [TestMethod]
+        public void VerifyVisualTreeForControlsInCommonStyles()
+        {
+            if(PlatformConfiguration.IsOSVersionLessThan(OSVersion.NineteenH1))
+            {
```

Comment:
What about the verification requires 19H1?

Reply:
We are using conditional xaml for some controls so the master files may be different for different versions. I've removed it after testing on all versions and updating master files (same for the "-8" in master file names).

Reply:
Great!